### DISCUSSION TITLE
netdata - no username and password option

### DISCUSSION THREAD

**Discussion by iptvcld** [linuxserver/Heimdall#753] [App issues]
my netdata goes via reverse proxy htaccess and the config section does not have an option for username and password

**Comment by KodeStar** [comment]
Stick it in the url `http://username:password@example.com/`

**Comment by iptvcld** [comment]
Problem is locking the console when you leave your system. A lock button is
needed. For now I have created a user name Lock and when I want to lock my
screen, I just press switch user and it locks me.
